src/main/steps/birth-father/manual-address/content.ts

```diff
@@ -0,0 +1,56 @@
+import { TranslationFn } from '../../../app/controller/GetController';
```

Comment:
can we group together the address related routes so as to be inline with other part of code
/address/lookup
/address/select
/address/manual
/address/international